pkg/cmd/prebuild/update.go

```diff
@@ -43,108 +52,131 @@ var prebuildUpdateCmd = &cobra.Command{
 			return nil
 		}
 
-		if len(args) < 2 {
-			var prebuilds []apiclient.PrebuildDTO
-			var res *http.Response
-			var selectedProjectConfigName string
+		if branchFlag != "" && retentionFlag != 0 {
```

Comment:
Why are we only checking these 2 flags here?

Reply:
Modified in the current updated commit